google-sheets/.env

```diff
@@ -0,0 +1,20 @@
+# description: A JSON key file for your Google Sheets service account
+# format: file(json)
+# contentKey: SHEETS_AUTH_JSON_CONTENT
+# link: https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys
+# required: true
+SHEETS_AUTH_JSON=/assets/auth.private.json
```

Comment:
This doesn't work for the reason I outlined in a comment below. It should be `/auth.json` and Quick Deploy has to figure out that that's `/assets/auth.private.json`.

Reply:
Updated to `/auth.json`.